src/services/redis.ts

```diff
@@ -0,0 +1,11 @@
+import { RedisOptions } from 'ioredis';
+// tslint:disable-next-line:no-var-requires
```

Comment:
It's better to use `import` instead of `require` but if it has problem with `import` and you want disable eslint , you had to disable it with `eslint` comments not `tslint` , because we use eslint in this project, not eslitn